src/components/InlineError/README.md

```diff
@@ -82,6 +82,24 @@ Use when the merchant has entered invalid information into multiple fields insid
 
 <!-- /content-for -->
 
+<!-- content-for: web -->
+
+---
+
+## Accessibility
+
+### Structure
```

Comment:
This example gives context for HTML output that users may not be familiar with, and provides rationale for using Inline errors instead of some other text type. I also called out a visual design choice that aids accessibility to help dissuade consumers from hiding the icon.